sentences/de/cover_HassCloseCover.yaml

```diff
@@ -2,4 +2,6 @@ language: de
 intents:
   HassCloseCover:
     data:
-      - sentences: []
+      - sentences:
+          - "schließe <area> <cover_variants>"
+          - "schließe <cover_variants> <area>"
```

Comment:
If you are adding the "(mach | mache) ... <turn_on/turn_off>" variants to other domains, I suggest also adding this to the covers, e.g. "mache die Garage zu".
Even if "mache <something>" is not the highest form of German, I guess it it used quite often.